scss/detail.scss

```diff
@@ -52,3 +53,15 @@
   width: 540px;
   border-bottom: 4px solid $earls-green;
 }
+
+.book-badge {
+  -ms-transform: rotate(15deg);
+  -webkit-transform: rotate(15deg);
+  bottom: 276px;
+  transform: rotate(15deg);
+  left: 210px;
+  position: absolute;
+  right: 0;
```

Comment:
You should only use one. Right or Left. The same applies for top and bottom.

Also, I'd use Top & Right so you can have smaller values. When using such big values it's hard to know how its behavior will be.

Reply:
Yes, this it's a mistake